app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,73 @@
+<p style="color: green"><%= notice %></p>
+
+<section>
+  <h2 class='recipe-name'><%= @recipe.name %></h2>
+
+  <div class='recipe-main'>
+    
+    <div class='public-btn-div'>
+      <p class='sub-title'>Preparation time : <%= @recipe.preparation_time %></p>
+      <div class='public-btn-flex'>
+        <p class='sub-title'>Public :</p>
+        <%= button_to  recipe_path(@recipe.id), :method => :patch do %>
+          <% if @recipe.public %>
+          <% end %>
+          <% if !@recipe.public %>
+        
+          <% end %>
+          <% end %>
+      </div>
```

Comment:
- [ ] Users don't have any feedback about the public status of the recipe, all I see is an empty button:

![image](https://github.com/gsmalik030/recipe_app/assets/77069011/2cf06a99-12ac-4bc1-9102-fcb6d7607833)

When I click it, I get this error:

![image](https://github.com/gsmalik030/recipe_app/assets/77069011/c72d326b-2e6e-4348-9481-38e8de96dddb)

Please consider fixing this! 🤓 🔧